bin/main.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+puts "Hello Word!"
```

Comment:
Running `bin/main.rb` does not execute your main file.

Are you sure you did the right thing?

Refer to [this](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) to refresh you memory.

Also, remember to end your files with a new line.